src/utils/encodePassword.ts

```diff
@@ -0,0 +1,11 @@
+import * as argon2 from 'argon2';
+
+export const encodePassword = async (rawPassword: string) => {
+  const SALT_ROUNDS = Number(process.env.SALT_ROUNDS) || 10;
+  return await argon2.hash(rawPassword, {
```

Comment:
**suggestion**: Consider making `memoryCost` and `parallelism` configurable via environment variables for more flexibility.